dev-packages/e2e-tests/test-applications/nestjs-basic/src/app.controller.ts

```diff
@@ -68,4 +69,9 @@ export class AppController {
   async killTestCron() {
     this.appService.killTestCron();
   }
+
+  @Get('flush')
+  async flush() {
+    flush();
```

Comment:
m: It may be worth to await the flush() call. Then we can be sure when we await the fetch call, that the flush went fully through.